src/widgets/settingspages/KeyboardSettingsPage.cpp

```diff
@@ -26,6 +26,8 @@ KeyboardSettingsPage::KeyboardSettingsPage()
     form->addRow(new QLabel("Ctrl + Shift + T"), new QLabel("Create new tab"));
     form->addRow(new QLabel("Ctrl + Shift + W"),
                  new QLabel("Close current tab"));
+    form->addRow(new QLabel("Ctrl + H"),
```

Comment:
Remove

Reply:
as stated on comment above in window.cpp